frontend/.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
To ne dat v ignore,  na podlagi tega ti npm namesti identične pakete, saj so vse odvisnosti definirane v detajle